### PR TITLE
Add HDZero goggle page

### PR DESCRIPTION
**Pull request by pkendall64**
Adds HDZero goggle backpack setup information.

### CODE REVIEW

File: docs/hardware/backpack/hdzero-goggles.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+template: main.html
+description: How to setup, flash and update your HDZero Goggles ExpressLRS Backpack.
+---
+
+![Hardware-Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/hardware.png)
+
+## Flashing the HDZero Goggles Backpack:
+
+After building the VRx firmware, copy the follwing 4 files into a folder named `ELRS` on the root of the SD card.
```
Comment (deadbytefpv) [inline docs/hardware/backpack/hdzero-goggles.md:10]: *follwing

Can we elaborate where these 4 files are? "Build" would only show the 2 bin files currently, and none of the other files.
C:\.expresslrs-configurator\firmwares\github\Backpack\.pio\build\<target name>\ would usually have these files (at least 2 on current available targets).

File: docs/hardware/backpack/hdzero-goggles.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+![Uploading](../../assets/images/hdzero/uploading.png)
+
+When the upload has completed the page will display the version (or branch) of backpack firmware that is installed next to the update button.
+
+![Complete](../../assets/images/hdzero/uploadcomplete.png)
+
+## Starting Backpack WiFi
+
+To start the WiFi endpoint on the backpack, navigate to the `Connections` page of the goggles UI and select the `Start Backpack WiFi` option. If you have configured your backpack with you home WiFi credentials, it will attempt to connect to that and will fallback to creating an access point, with name `ExpressLRS VRx Backpack` and password `expresslrs`. The web UI will be available as `http://elrs_vrx` or `http://elrs_vrx.local`
```
Comment (deadbytefpv) [inline docs/hardware/backpack/hdzero-goggles.md:36]: "and will fallback to creating an access point,"
should this be:
"and will fallback to creating an access point if it cannot connect,"

File: docs/hardware/backpack/hdzero-goggles.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Binding to Your Transmitter
+
+!!! info "Attention"
+    Since the goggles have a bind button in the `Connections` page in the UI, the option to enter binding mode by power-cycling 3x to bind has been disabled.
+
+If you entered teh same bind-phrase as on your TX module and TX backpack when building the goggle backpack firmware then your device will already be bound and you can validate this looking at the UID next to the `Start Backpack Binding` button on the `Connections` page in teh goggle UI.
```
Comment (deadbytefpv) [inline docs/hardware/backpack/hdzero-goggles.md:43]: *teh

File: docs/hardware/backpack/hdzero-goggles.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+If you have NOT entered your bind-phraase into ExpressLRS Configurator when building your backpack firmware, you can use the `Start Backpack Binding` option in the `Connections` page in the UI. Once you have selected this option the goggles will wait for a bind message from your transmitter. To perform the bind from the transmitter, enter the Lua script and sleect the `[Bind]` option there. When the message is sent and the goggles have reeived the bind message, they should display `Success` in place of the button text. If they do not receive a bind message then they will timeout after 2 minutes and display a `Timeout` message.
```
Comment (deadbytefpv) [inline docs/hardware/backpack/hdzero-goggles.md:45]: *phraase

*enter the Lua script and sleect (select)
can we rephrase this part to:
"load the ExpressLRS Lua Script on your radio handset and select..."

*reeived - received

**Review by deadbytefpv — CHANGES_REQUESTED** [review]

File: docs/hardware/backpack/hdzero-goggles.md (hunk 1 above)
Comment (pkendall64) [reply docs/hardware/backpack/hdzero-goggles.md]: Spelling fixed.
I need to raise a ticket for configurator to copy the 4 files to the temporary folder that it opens.

File: docs/hardware/backpack/hdzero-goggles.md (hunk 2 above)
Comment (pkendall64) [reply docs/hardware/backpack/hdzero-goggles.md]: done

File: docs/hardware/backpack/hdzero-goggles.md (hunk 4 above)
Comment (pkendall64) [reply docs/hardware/backpack/hdzero-goggles.md]: done

File: docs/hardware/backpack/hdzero-goggles.md (hunk 3 above)
Comment (pkendall64) [reply docs/hardware/backpack/hdzero-goggles.md]: done

File: docs/hardware/backpack/hdzero-goggles.md (hunk 1 above)
Comment (pkendall64) [reply docs/hardware/backpack/hdzero-goggles.md]: Created a ticket for the configurator guys https://github.com/ExpressLRS/ExpressLRS-Configurator/issues/479
Thats one app I can't get my head around to create a PR 🤪

**Review by deadbytefpv — APPROVED** [review]

**Review by maybenikhil — APPROVED** [review]

**Pull request merged**